lib/public_key/src/pubkey_os_cacerts.erl

```diff
@@ -171,11 +171,25 @@ load_win32() ->
     store(lists:foldl(Dec, [], os_cacerts())).
 
 load_darwin() ->
+    SystemRootsKeyChainFile = "/System/Library/Keychains/SystemRootCertificates.keychain",
+    case get_darwin_certs(SystemRootsKeyChainFile) of
+         {ok, Bin1} ->
+            SystemKeyChainFile = "/Library/Keychains/System.keychain",
+            case get_darwin_certs(SystemKeyChainFile) of
+                 {ok, Bin2} ->
+                    decode_result(<<Bin1/binary, Bin2/binary>>);
+                  Err ->
```

Comment:
Can this fail?   
And if so should we just decode the SystemRoot bin which we already got and return that part which was successful?

Reply:
I thought about this, the system keychain is available to all users, so if it did fail, your system has serious problems :) That said, if we wanted to proceed anyway, we could log an error and proceed, wdyt?

Reply:
I thought that better safe than sorry, that will keep backward compatibility and since we can't test it, it might be a good idea.   And I don't know anything MacOS so I don't know if it always is available on all versions.

Reply:
It's a solid point, I also want to take a stab at testing this on GH, the runner may act differently than my laptop 😄  No reason to rush this.